Resources/Prototypes/Entities/Markers/atmos_blocker.yml

```
@@ -113,11 +113,20 @@
+      layers:
+        - sprite: Markers/atmos.rsi # {
+          state: base
+          shader: unshaded
+        - sprite: Markers/atmos.rsi
+          shader: unshaded # }
+          state: nitrogen
+    - type: AtmosFixMarker
+      gasMix:
+        volume: 2500
+        moles:
+          - 0 # oxygen
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)